packages/headless/src/features/history/history-state.ts

```diff
@@ -37,6 +38,8 @@ export function extractHistory(state: Partial<HistoryState>): HistoryState {
     dateFacetSet: state.dateFacetSet || getDateFacetSetInitialState(),
     categoryFacetSet:
       state.categoryFacetSet || getCategoryFacetSetInitialState(),
+    automaticFacetSet:
+      state.automaticFacetSet ?? getAutomaticFacetSetInitialState(),
```

Comment:
🤔 curious, why nullish here? (wondering because all others are ||)

Reply:
Nullish is the right operator in this case. I think the others were made before the nullish operator came out or was better known. Sonarlint tells me to change the '||' to '??', that's why I used the nullish operator instead.